static/app/views/stories/useStoriesLoader.tsx

```diff
@@ -1,69 +1,28 @@
-import {useCallback, useEffect, useState} from 'react';
-
+import {useQuery} from 'sentry/utils/queryClient';
 import storiesContext from 'sentry/views/stories/storiesContext';
 import type {ResolvedStoryModule} from 'sentry/views/stories/types';
 
-interface Props {
+interface UseStoriesLoaderOptions {
   filename: string;
 }
 
-interface EmptyState {
-  error: undefined;
-  filename: undefined | string;
-  resolved: undefined;
-}
-
-interface ResolvedState {
-  error: undefined;
+interface StoryDescriptor {
```

Comment:
I introduced this type as otherwise the index signature of ResolvedStoryModule satisfies the filename and resolved signature and the type error is not caugh